packages/components/organisms/f-checkout/src/services/tests/placeOrderApi.test.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+import placeOrderApi from '../orderPlacementApi';
+import { version as applicationVersion } from '../../../package.json';
+import { mockAuthToken } from '../../components/_tests/helpers/setup';
+
+const authToken = mockAuthToken;
+
+describe('BasketApi', () => {
+    let payload;
+    let config;
+    let state;
+
+    beforeEach(() => {
+        // Arrange
+        payload = {
+            url: 'http://localhost/account/checkout',
+            timeout: 10000,
+            data: ''
+        };
+
+        state = {
+            authToken
+        };
+
+        config = {
+            headers: {
+                'Content-Type': 'application/json;v=2',
+                'x-je-application-id': 7, // Responsive Web
+                'x-je-application-version': applicationVersion,
+                Authorization: `Bearer ${authToken}`
+            },
+            timeout: payload.timeout
+        };
+
+        axios.post = jest.fn(() => Promise.resolve({ status: 200 }));
+    });
+    describe('getBasket ::', () => {
+        it('should get the basket details from the backend', async () => {
```

Comment:
Test names are wrong here. Copy-pasting, it seems 😬

Reply:
🤦 oh dear!